armi/reactor/converters/uniformMesh.py

```diff
@@ -390,12 +390,13 @@ def adjointFluxGetter(block, _paramNames):
             _setStateFromOverlaps(
                 aSource, aDest, paramSetter, paramGetter, self.blockParamNames
             )
-            # Now recalculate derived params with the mapped flux to minimize
-            # potential numerical diffusion (e.g. control rod tip into large coolant)
-            for b in aDest:
-                globalFluxInterface.calcReactionRates(
-                    b, destReactor.core.p.keff, destReactor.core.lib
-                )
+            if destReactor.core.lib is not None:
+                # Now recalculate derived params with the mapped flux to minimize
+                # potential numerical diffusion (e.g. control rod tip into large coolant)
```

Comment:
Why it's okay not to do this if there is no lib does make sense to explain. "if we don't have a lib, then XS haven't been provided or calculated, stands to reason that no reaction rates/flux has been calculated either"